x/gravity/keeper/keeper.go

```diff
@@ -239,3 +242,82 @@ func (gsg GravityStoreGetter) Store(ctx context.Context) storetypes.KVStore {
 func (k Keeper) GetAuthority() string {
 	return k.authority
 }
+
+func (k Keeper) BridgeTax(ctx context.Context) (*types.BridgeTax, error) {
+	val := k.GetStore(ctx, types.StoreModulePrefix).Get(types.BridgeTaxKey)
+
+	if len(val) == 0 {
+		// We have no bridge tax settings
+		return nil, nil
+	}
+
+	var tax types.BridgeTax
+	if err := k.cdc.Unmarshal(val, &tax); err != nil {
+		return nil, err
+	}
+
+	return &tax, nil
+}
+
+func (k Keeper) SetBridgeTax(ctx context.Context, tax *types.BridgeTax) error {
+	taxRate, ok := new(big.Rat).SetString(tax.Rate)
```

Comment:
It should be fine. If it is an integer, it gets serialized as a `big.Int`, otherwise it's serialized as `<numerator>/<denominator>` where numerator and denominator are both integers.

But we can keep `tax.Rate` serialized as string in the store and keep this implementation separate, so we won't have an issue replacing `bit.Rat` with something else in the future if we want to.